### PR TITLE
planner: enable revive

### PR DESCRIPTION
**Pull request by hawkingrei**
Signed-off-by: Weizhen Wang <wangweizhen@pingcap.com>

### What problem does this PR solve?

Issue Number: close #xxx

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by tangenta — APPROVED** [review]

File: planner/core/explain.go [hunk 1]
```diff
@@ -429,6 +431,7 @@ func (p *PhysicalIndexJoin) explainInfo(normalized bool, isIndexMergeJoin bool)
 		for i := range p.OuterHashKeys {
 			expr, err := expression.NewFunctionBase(MockContext(), ast.EQ, types.NewFieldType(mysql.TypeLonglong), p.OuterHashKeys[i], p.InnerHashKeys[i])
 			if err != nil {
+				log.Warn("NewFunctionBase", zap.Error(err))
```
Comment (time-and-fate) [inline planner/core/explain.go:434]: Why we are not using `logutil.BgLogger` like other places?

Comment (time-and-fate) [reply planner/core/explain.go:434]: I think we can add some information to help to locate this if it appears in the log, like the method name "(*PhysicalIndexJoin).explainInfo".

Comment (hawkingrei) [reply planner/core/explain.go]: > Why we are not using `logutil.BgLogger` like other places?

Ok, I have updated.

**Pull request merged**